features/FxA-89-devices-view/README.md

```diff
@@ -0,0 +1,411 @@
+
+# Devices View
+
+Feature card: https://github.com/mozilla/fxa-features/issues/8
+
+Original draft: https://docs.google.com/document/d/1VrysJ4LFPZ20IeMMmRts5l_2GUHhm890rlQhD012U_g
+
+Engineering bugs: https://waffle.io/mozilla/fxa?milestone=FxA-89:%20devices%20view
+
+## Stories
+
+* So that I can best ensure the integrity of my Sync setup,
+  as a Firefox Sync user,
+  I want to see information about the devices
+  connected to my Firefox Account.
+* So that I can obtain a device overview more quickly,
+  I want the devices connected to my account
+  to have recognizable names by default.
+* I want to easily see the type
+  and browser details of the each device, 
+* So that I can better tell what device I am currently using,
+  I want it to be easily distinguishable from the other items.
+* So that I can easily detect when a device is no longer active,
+  I want to be able to see when my devices last authenticated.
```

Comment:
As per https://github.com/mozilla/fxa-auth-server/issues/1438#issuecomment-243934547, should we add a story to cover distinguishing between confirmed/unconfirmed devices?